index.js

```diff
@@ -151,24 +151,10 @@ function getWebGLContext(failIfMajorPerformanceCaveat) {
     var attributes = Object.create(isSupported.webGLContextAttributes);
     attributes.failIfMajorPerformanceCaveat = failIfMajorPerformanceCaveat;
 
-    if (canvas.probablySupportsContext) {
-        return (
-            canvas.probablySupportsContext('webgl', attributes) ||
-            canvas.probablySupportsContext('experimental-webgl', attributes)
-        );
-
-    } else if (canvas.supportsContext) {
-        return (
-            canvas.supportsContext('webgl', attributes) ||
-            canvas.supportsContext('experimental-webgl', attributes)
-        );
-
-    } else {
-        return (
-            canvas.getContext('webgl', attributes) ||
```

Comment:
Are there any cases where calling `canvas.getContext` could throw an exception if webgl were unsupported on the user's browser?

Reply:
Hopefully not. MDN says

 > The HTMLCanvasElement.getContext() method returns a drawing context on the canvas, or null if the context identifier is not supported.